src/Form/OgMenuForm.php

```diff
@@ -7,15 +7,15 @@
 
 namespace Drupal\og_menu\Form;
 
-use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Entity\BundleEntityFormBase;
 use Drupal\Core\Form\FormStateInterface;
 
 /**
  * Class OgMenuForm.
  *
  * @package Drupal\og_menu\Form
  */
-class OgMenuForm extends EntityForm {
+class OgMenuForm extends BundleEntityFormBase {
   /**
```

Comment:
One blank line should be added. Also the @file comment at the top of the file is not required.